test/parallel/test-performance-measure.js

```diff
@@ -10,7 +10,9 @@ const ALLOWED_MARGIN = 10;
 const expected = ['Start to Now', 'A to Now', 'A to B'];
 const obs = new PerformanceObserver(common.mustCall((items) => {
   items.getEntries().forEach(({ name, duration }) => {
-    assert.ok(duration > (DELAY - ALLOWED_MARGIN));
+    // TODO(mrjithil) remove after 55431 fixed
```

Comment:
Nit: it would be more helpful for the `55431` to provide a complete URL. It would also be helpful to expand a bit with a quick summary of what that issue is.